notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -10,7 +10,7 @@
 
 define([
     'base/js/utils',
-    'base/js/keyboard',
+    'base/js/keyboard'
```

Comment:
Stripping trailing commas makes me a little sad. Just curious: what brought this on?